apm-commons/apm-datacarrier/pom.xml

```diff
@@ -26,5 +26,17 @@
     <modelVersion>4.0.0</modelVersion>
 
     <artifactId>apm-datacarrier</artifactId>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>8</source>
+                    <target>8</target>
```

Comment:
The common lib can't compile in 1.8, because the agent should be jdk 1.6 compatible.

Reply:
I have used Lambda in unit test, will try to change with unit test and make it compatible with JDK 1.6